src/main/resources/application.yml

```diff
@@ -40,3 +40,5 @@ logging:
 azure:
   app_insights_key: ${APPINSIGHTS_INSTRUMENTATIONKEY:false}
 
+liquibase:
```

Comment:
Better to exclude it from the jar in the first place

Reply:
may need it locally